modules/network-flow-matrix.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/configuring-firewall.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-flow-matrix_{context}"]
+= {product-title} network flow matrix
+
+The network flow matrix describes the ingress flows to {product-title} services.
+Use the information in the network flow matrix to help you manage ingress traffic.
+
+To view or download the raw CSV content, see link:https://raw.githubusercontent.com/rohennes/openshift-docs/TELCODOCS-1670/snippets/network-flow-matrix.csv[this resource].
+
+[NOTE]
+====
+The network flow matrix describes common ingress traffic flows. It does not describe every possible flow. Some flows that are not described in this matrix might be critical to operation. For example, if you block all traffic and then selectively open only the flows described here, you might unintentionally block a necessary flow.
```

Comment:
I dont think this note is accurate.
the whole plan is to support this matrix
so it is accurate for base openshift.
what we do want to say, is that it does not encompass optional component (e.g. operators from market place)

Reply:
I've updated the content accordingly